client/src/main.rs

```diff
@@ -5,19 +5,35 @@ use std::sync::Arc;
 use anyhow::{bail, Result};
 use bytes::{BufMut, Bytes, BytesMut};
 use rand::prelude::*;
+use rand_chacha::rand_core::SeedableRng;
+use structopt::clap::arg_enum;
 use structopt::StructOpt;
 use tokio::runtime::Builder;
 
 use crucible::*;
 
+/*
+ * The various tests this program supports.
+ */
+arg_enum! {
+    #[derive(Debug, StructOpt)]
+    enum Workload {
+        One,
+        Big,
+        Dep,
+        Rand,
+        Baloon,
```

Comment:
typo

Reply:
fixed in 78fd508

And, perhaps it's time for me to return to 7th grade and finish my spelling assignments.